packages/ag-charts-enterprise/src/box-plot/boxPlotSeries.ts

```diff
@@ -0,0 +1,319 @@
+
+    // @ts-ignore
+    getTooltipHtml(seriesDatum: any): string {
+        return '';
+    }
+
+    protected isLabelEnabled(): boolean {
+        return false;
+    }
+
+    protected async updateDatumSelection(opts: {
+        nodeData: BoxPlotNodeDatum[];
+        datumSelection: _Scene.Selection<_Scene.Group, BoxPlotNodeDatum>;
+    }) {
+        const { nodeData, datumSelection } = opts;
+        const data = nodeData ?? [];
+        return datumSelection.update(
+            data.filter(
+                (datum) =>
+                    datum.minValue <= datum.q1Value &&
+                    datum.q1Value <= datum.medianValue &&
+                    datum.medianValue <= datum.q3Value &&
+                    datum.q3Value <= datum.maxValue
+            )
+        );
+    }
+
+    protected async updateDatumNodes(opts: {
+        datumSelection: _Scene.Selection<_Scene.Group, BoxPlotNodeDatum>;
+        isHighlight: boolean;
+    }) {
+        const { datumSelection } = opts;
+
+        datumSelection.each((group, datum) => {
+            const selection = _Scene.Selection.select(group, _Scene.Rect);
+            const boxes = selection.selectByTag<_Scene.Rect>(GroupTags.Box);
+            const whiskers = selection.selectByTag<_Scene.Line>(GroupTags.Whisker);
+            const caps = selection.selectByTag<_Scene.Line>(GroupTags.Cap);
+
+            // TODO cleanup; preferably with generic methods
+            boxes[0].x = datum.q1Value;
+            boxes[0].y = datum.yValue;
+            boxes[0].width = datum.medianValue - datum.q1Value + datum.strokeWidth;
+            boxes[0].height = datum.bandwidth;
+
+            boxes[1].x = datum.medianValue;
+            boxes[1].y = datum.yValue;
+            boxes[1].width = datum.q3Value - datum.medianValue;
+            boxes[1].height = datum.bandwidth;
+
+            caps[0].x1 = caps[0].x2 = datum.minValue;
+            caps[1].x1 = caps[1].x2 = datum.maxValue;
+            caps[0].y1 = caps[1].y1 = datum.yValue + datum.bandwidth * 0.25;
+            caps[0].y2 = caps[1].y2 = datum.yValue + datum.bandwidth * 0.75;
+
+            whiskers[0].x1 = datum.minValue;
+            whiskers[0].x2 = datum.q1Value;
+            whiskers[1].x1 = datum.q3Value;
+            whiskers[1].x2 = datum.maxValue;
+            whiskers[0].y1 = whiskers[0].y2 = whiskers[1].y1 = whiskers[1].y2 = datum.yValue + datum.bandwidth * 0.5;
+
+            boxes[0].fill = boxes[1].fill = datum.fill;
+            boxes[0].stroke =
+                boxes[1].stroke =
+                whiskers[0].stroke =
+                whiskers[1].stroke =
+                caps[0].stroke =
+                caps[1].stroke =
+                    datum.stroke;
+            boxes[0].strokeWidth =
+                boxes[1].strokeWidth =
+                whiskers[0].strokeWidth =
+                whiskers[1].strokeWidth =
+                caps[0].strokeWidth =
+                caps[1].strokeWidth =
+                    datum.strokeWidth;
```

Comment:
Nit: I would avoid these chained assignments, it doesn't follow our typical coding pattern.